src/observer.test.ts

```
@@ -44,6 +42,21 @@ const OneMiB = 1048576;
 
 const entropy = Buffer.from('entropy');
 
+function createReport(
+  overrides: Partial<ObserverReport> & Pick<ObserverReport, 'gatewayAssessments'>,
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)